janitor/functions.py

```diff
@@ -3869,3 +3813,230 @@ def sort_naturally(
     """
     new_order = index_natsorted(df[column_name], **natsorted_kwargs)
     return df.iloc[new_order, :]
+
+
+@pf.register_dataframe_method
+def expand_grid(df: pd.DataFrame = None,
+                df_key: str = None,
+                others: Dict = None) -> pd.DataFrame:
+    """ Creates a dataframe from all combinations of all inputs.
+        This method mutates the original dataframe.
```

Comment:
Minor formatting suggestion.

```suggestion
    """
    Creates a dataframe from all combinations of all inputs.
```

Also removed the mutation comment - I don't believe this mutates the original, but returns a new dataframe. ("Mutation" in the pyjanitor docs refers to whether an object is modified in place or not.)